content_scripts/link_hints.coffee

```diff
@@ -125,6 +125,22 @@ LinkHints =
     marker
 
   #
+  # Find elements that have delegated onclick event listener assigned,
+  # and mark them with `vimium-has-delegated-onclick-listener` attribute.
+  # This will make sure those elements are discovered during `getVisibleClickableElements` call.
+  #
+  markElementsThatHaveDelegatedOnClickListener: ->
+    for element in document.querySelectorAll("*[vimium-jquery-delegated-events-selectors]")
+      selectorsStr = element.getAttribute "vimium-jquery-delegated-events-selectors"
+      continue unless selectorsStr
+
+      selectors = selectorsStr.split("|").filter (x) -> !!x
+
+      for selector in selectors
+        for child in element.querySelectorAll(selector)
```

Comment:
> it seems like we cannot access jQuery, defined on the page, from the content script... Is that right, @mrmr1993?

Unfortunately it is. I would send a custom event from the content script and mark the elements we want when we catch it in the injected code. Obviously then we can resume code execution in link hints with another custom event.

Since a lot of the time we don't have to do this, we should also only send the event when we have  a `querySelector` match on the delegated event selector.